FinSearchUnified/Resources/services.xml

```diff
@@ -16,6 +16,12 @@
             <tag name="shopware.event_subscriber" />
         </service>
 
+        <service id="fin_search_unified.sdym_criteria_request_handler"
```

Comment:
Put the attribute `id` on a separate line and move both attributes one level to the left (like the other service definitions).